src/lib/txVerify.js

```diff
@@ -17,6 +17,59 @@ const TransactionInconsistentReason = {
   RECIPIENT_CRYPTO_ADDRESS_MISMATCH: 'recipient_crypto_address_mismatch'
 }
 
+export async function checkTxInProcess(api, context, withoutNonce = true, externalNonce = null) {
+  const fetchInfo = CryptosInfo[context.state.crypto].txFetchInfo
+  const TX_LOCK_TIME = fetchInfo.newPendingAttempts * fetchInfo.newPendingInterval
+
+  const currentNonce = context.rootGetters.getTransactionInProcess(context.state.crypto)
+
+  const expiration = Date.now()
```

Comment:
The name is confusing. I suggest to pick another name, e.g. `today`, `currentTimestamp`, or any other more meaningful name at your choice.